apps/docs/src/.vitepress/components/DesignVariable.vue

```diff
@@ -54,6 +55,9 @@ const emit = defineEmits<{
 
     <OnyxIcon v-else class="variable__copy" :icon="copyIcon" color="primary" />
   </button>
+  <div v-if="props.value && props.type === 'value'" class="variable__value">
```

Comment:
In my opinion when looking at the design, the value should be part of the card component instead of the variable to keep things separated :) E.g. on the colors page its not needed